validation-test/compiler_crashers_fixed/28659-isactuallycanonicalornull-forming-a-cantype-out-of-a-non-canonical-type.swift

```diff
@@ -5,7 +5,6 @@
 // See https://swift.org/LICENSE.txt for license information
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
-// REQUIRES: asserts
-// REQUIRES: deterministic-behavior
-// RUN: not --crash %target-swift-frontend %s -emit-ir
+// RUN: not %target-swift-frontend %s -emit-ir
 {func b(UInt=1 + 1 as?Int){f
+
```

Comment:
I blame `resolve-crashes` 😇 

In reality it changes nothing.  The important thing in this crash is that we typecheck b as a function and look into its parameter list to validate types.